tests/test_wayne_local_emergency_planning.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.wayne_local_emergency_planning import WayneLocalEmergencyPlanningSpider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "wayne_local_emergency_planning.html"),
+    url="https://www.waynecounty.com/departments/hsem/wayne-county-lepc.aspx",
+)
+spider = WayneLocalEmergencyPlanningSpider()
+
+freezer = freeze_time("2019-10-03")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+
+def test_tests():
```

Comment:
Usually we'll want to either remove this entirely or add something like `test_count` that checks how many meetings are returned. That would make sense here, and we could make sure `parsed_items` has a length of 4

Reply:
I think I covered that with my last test:

def test_correct_number_of_items():
    assert len(parsed_items) == 4